cli/endpoints/batch/deploy-models/openai-embeddings/deploy-and-run.sh

```diff
@@ -0,0 +1,77 @@
+set -e
+
+# <set_variables>
+export ENDPOINT_NAME="<YOUR_ENDPOINT_NAME>"
+# </set_variables>
+
+# <set_openai>
+OPENAI_API_BASE="https://<deployment>.openai.azure.com/"
+# </set_openai>
+
+# <name_endpoint>
+ENDPOINT_NAME="text-davinci-002"
+# </name_endpoint>
+
+# The following code ensures the created deployment has a unique name
+ENDPOINT_SUFIX=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w ${1:-5} | head -n 1)
+ENDPOINT_NAME="$ENDPOINT_NAME-$ENDPOINT_SUFIX"
+
+echo "Register the model"
+# <register_model>
+MODEL_NAME='text-embedding-ada-002'
+az ml model create --name $MODEL_NAME --path "model"
+# </register_model>
+
+echo "Creating batch endpoint $ENDPOINT_NAME"
+# <create_endpoint>
+az ml batch-endpoint create -n $ENDPOINT_NAME -f endpoint.yml
+# </create_endpoint>
+
+echo "Creating batch deployment $DEPLOYMENT_NAME for endpoint $ENDPOINT_NAME"
+# <create_deployment>
+az ml batch-deployment create --file deployment.yml \
+                              --endpoint-name $ENDPOINT_NAME \
+                              --set-default \
+                              --set settings.environment_variables.OPENAI_API_BASE=$OPENAI_API_BASE
```

Comment:
How is this used in the batch job?

Reply:
Nvm. Sync'ed with Facundo offline. These are picked up by OpenAI automatically.

Reply:
`OPENAI_API_BASE` is the URL of the deployment you want to use. They are publicly documented by the OpenAI SDK and users are expected to know about them.